### DISCUSSION TITLE
KeyCloak issue

### DISCUSSION THREAD

**Discussion by rohandixitdelhi95** [Sunbird-cQube/community#830] [General]
Keycloak has been restarted but still after clicking on admin console its throwing an error.
![f0c76b27-b33e-4241-8916-f880d2fd2a92](https://github.com/Sunbird-cQube/community/assets/140483814/16e6d161-636e-47a8-aad3-f3d5d806c241)
![efad5c58-df80-4231-a726-97e5fdfc7655](https://github.com/Sunbird-cQube/community/assets/140483814/c8cd34f3-acc5-4150-87f8-c149359fe0f1)

**Comment by RNarmatha** [comment]
@Bommalla - Please look into this task once NVSK task is completed

**Comment by bommalla** [comment]
@rohandixitdelhi95 Can you please send me the keycloak logs.

sudo docker logs keycloak_app.

**Reply by rohandixitdelhi95** [reply]
Here I'm attaching the photos of logs.
![4a18288e-37ef-407c-932f-80f661856bb2](https://github.com/Sunbird-cQube/community/assets/140483814/af200de8-9fc6-4a5a-afcd-a33863badbce)

**Comment by bommalla** [comment]
@rohandixitdelhi95 Thanks,I'll check and get back to you

**Reply by rohandixitdelhi95** [reply]
@bommalla Ok .

**Comment by bommalla** [comment]
@rohandixitdelhi95 Can we connect at 3PM today ?

**Reply by rohandixitdelhi95** [reply]
Sure

**Comment by bommalla** [comment]
@rohandixitdelhi95 Issue resolved.

**Comment by RNarmatha** [comment]
Issue got resolved
